src/olympia/addons/models.py

```diff
@@ -1043,33 +1043,35 @@ def get_default_icon_url(self, size):
         )
 
     @write
-    def update_status(self, ignore_version=None):
+    def update_status(self, ignore_version=None, force_update=False):
```

Comment:
You never pass this argument. IMHO we should never force the status - if tests require it we should modify their versions so that it doesn't happen, or just keep using `addon.update(status=xxx)` for the special edge cases we want to test.